src/lib.rs

```diff
@@ -51,12 +51,14 @@ pub struct VelloSceneBundle {
     /// A transform to apply to this scene
     pub transform: Transform,
     /// User indication of whether an entity is visible. Propagates down the entity hierarchy.
-    pub view_visibility: ViewVisibility,
+    pub visibility: Visibility,
+    /// A bucket into which we group entities for the purposes of visibility.
+    pub visibility_class: VisibilityClass,
 }
 
 /// A simple newtype component wrapper for [`vello::Scene`] for rendering.
 #[derive(Component, Default, Clone, Deref, DerefMut)]
-#[require(Transform, ViewVisibility)]
+#[require(Transform, Visibility, VisibilityClass)]
```

Comment:
Bevy sprite also has this:
https://github.com/bevyengine/bevy/blob/e9418b3845c1ffc9624a3a4003bde66a2ad6566a/crates/bevy_sprite/src/sprite.rs#L19